### PR TITLE
74 adding image of the timeline in planning page

### PR DESCRIPTION
**Pull request by LizGuzRam**
Hi Nami, 
I added the image of the planning timeline, curious to see what you think of it?

### CODE REVIEW

File: assets/files/planning_timeline.jpg
Comment (nsunami) [inline assets/files/planning_timeline.jpg:1]: Nice! I noticed that the file size for this image is 2 MB, which is a bit large for a webpage in my opinion. I resize the image a bit and attaching to the comment here:


planning_timeline

Do you think it is a good idea to replace the image with this?

**Review by nsunami — APPROVED** [review]
I had one suggestion to reduce the file size of the image, but I will leave it up to you! 👍

File: assets/files/planning_timeline.jpg
Comment (LizGuzRam) [reply assets/files/planning_timeline.jpg:1]: Oh yes absolutely! Thank you for the edit!

Comment (LizGuzRam) [reply assets/files/planning_timeline.jpg:1]: @nsunami has the new image been linked already? If so then I can just merge this one right?

**Pull request merged**